pages/advanced/erc-7579/tutorials/7579-tutorial.mdx

```diff
@@ -0,0 +1,194 @@
+```
+
+It will load a `PRIVATE_KEY` from `.env.local` (or generate one if it doesn’t exist) and then create the `publicClient`, `paymasterClient` and `bundlerClient` necessary to initialize the `permissionless` library. It then creates a 7579-enabled Safe Smart Account from the generated private key, before passing it to `createSmartAccountClient` to generate a Permissionless client.
+
+## 3. Add Rhinestone module functionality
+
+Create a new file `scheduledTransfers.ts` in the `lib` folder:
+
+```shell
+touch scheduledTransfers.ts
+```
+
+Add the code necessary to create a scheduled transfer using Rhinestone’s `ScheduledTransfers` module:
+```typescript
+// from ../../../../examples/erc-7579/lib/scheduledTransfers.ts
+```
+
+This file contains two functions:
+- `install7579Module` will install the module to a Safe and schedule its first transfer; 
+- `scheduleTransfer` to schedule transfers in a Safe where the module has been previously installed.
+
+## 4. Add simple UI
+
+Now that we have the logic necessary to set up a safe and schedule a transfer, let's create a simple UI to interact with it. Create a new file `ScheduledTransferForm.tsx` in the `components` folder:
+
+```shell
+cd ..
+mkdir components
+cd components
+touch ScheduledTransferForm.tsx
+```
+
+Add the following code to `ScheduledTransferForm.tsx`:
+
+```tsx
+// from ../../../../examples/erc-7579/components/ScheduledTransferForm.tsx
+```
+
+This component will detect whether the Safe has the module installed, and allow the user to input the amount, receiver address, and date and time for the scheduled transfer.
+
+Now, edit `app/page.tsx` to include the `ScheduledTransferForm` component:
+
+```tsx
+// from ../../../../examples/erc-7579/app/page.tsx
+```
+
+## 5. Add styling (optional)
+
+We can add some styling to our app by editing the contents of `layout.tsx` in `app` folder:
+
+```tsx
+// from ../../../../examples/erc-7579/app/layout.tsx
+```
+
+This will add some basic styling to the app, including a header. You can also add some custom CSS to `globals.css` in the same folder:
+
+```css
+h1,
+h2,
+h3 {
+  margin-top: 40px;
+  margin-bottom: 10px;
+}
+
+button {
+  cursor: pointer;
+  border: none;
+  background: #00e673;
+  color: black;
+  padding: 10px 20px;
+  border-radius: 5px;
+  margin: 10px 0;
+}
+
+input {
+  padding: 10px;
+  border-radius: 5px;
+  border: 1px solid #ccc;
+  margin: 10px 0;
+}
+
+button:disabled {
+  background: #ccc;
+  color: #666;
+}
+```
+
+## **Testing your Safe 7579 app**
+
+That's it! You can find the source code for the example created in this tutorial [on GitHub](https://github.com/5afe/safe-7559-tutorial). You can now return to your browser and see the app displayed 🎉.
+
+![screenshot-finalized](../../../../assets/7579-tutorial-2.png)
+
+Click the **Create Safe** button to initialize Permissionless' client with the private key you stored on `.env.local`. It will deploy an ERC-7579-enabled Safe on its first transaction.
```

Comment:
```suggestion
Click the **Create Safe** button to initialize Permissionless' client with the private key you stored on `.env.local` file. It will deploy an ERC-7579-compatible Safe Smart Account on its first transaction.
```